source/includes/fundamentals/code-snippets/bson-extra-elements/BsonExtraElementsExample.java

```diff
@@ -0,0 +1,74 @@
+package fundamentals;
+
+import static com.mongodb.MongoClientSettings.getDefaultCodecRegistry;
+import static org.bson.codecs.configuration.CodecRegistries.fromProviders;
+import static org.bson.codecs.configuration.CodecRegistries.fromRegistries;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.bson.codecs.configuration.CodecRegistry;
+import org.bson.codecs.pojo.ClassModel;
+import org.bson.codecs.pojo.Conventions;
+import org.bson.codecs.pojo.PojoCodecProvider;
+
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.model.Filters;
+import com.mongodb.client.model.Updates;
+import com.mongodb.client.result.UpdateResult;
+
+public class BsonExtraElementsExample {
+
+	private static String DOC_NAME_VALUE = "MDB0123";
+
+    private static CodecRegistry getCodecRegistry() {
+        ClassModel<ProductWithBsonExtraElements> classModel = ClassModel.builder(ProductWithBsonExtraElements.class).
+                conventions(Arrays.asList(Conventions.ANNOTATION_CONVENTION)).build();
```

Comment:
I think this convention is enabled by default.  Any reason to include it here explicitly?  i.e. you can just do

    PojoCodecProvider.builder().automatic(true).build();